### PR TITLE
changes by someacerclone

### PR DESCRIPTION
**Pull request by someacerclone**

### CODE REVIEW

**Review by someacer — COMMENTED** [review]
Some comment from someacer

**Review by someacer — CHANGES_REQUESTED** [review]
Please add more words.

**Comment by someacerclone** [comment]
more words are added

File: README.md [hunk 1]
```diff
@@ -7,4 +7,6 @@ Here are other changes
 
 Here are some more changes
 
+Here are changes by someacerclone
 
+Here are some more words you requested
```
Comment (someacerclone) [inline README.md:12]: here are some more words

Comment (someacer) [reply README.md:12]: Nice

**Review by someacer — APPROVED** [review]
More words added, approve

**Pull request merged**